mongonaut/urls.py

```diff
@@ -1,37 +1,77 @@
-from django.conf.urls import patterns, url
-
+from django.conf.urls import url
 from mongonaut import views
 
-urlpatterns = patterns('',
-    url(
-        regex=r'^$',
-        view=views.IndexView.as_view(),
-        name="index"
-    ),
-    url(
-        regex=r'^(?P<app_label>[_\-\w\.]+)/(?P<document_name>[_\-\w\.]+)/$',
-        view=views.DocumentListView.as_view(),
-        name="document_list"
-    ),
-    url(
-        regex=r'^(?P<app_label>[_\-\w\.]+)/(?P<document_name>[_\-\w\.]+)/add/$',
-        view=views.DocumentAddFormView.as_view(),
-        name="document_detail_add_form"
-    ),
-    url(
-        regex=r'^(?P<app_label>[_\-\w\.]+)/(?P<document_name>[_\-\w\.]+)/(?P<id>[\w]+)/$',
+urlpatterns = []
```

Comment:
To clean this up do you mind putting all the `url` creations in this list and then exploding them out in the call inside the `try/catch`? 

For example:

```python
urlpatterns = [
    url() # All the patterns in this list.
]

try:
    from django.conf.urls import patterns

    urlpatterns = patterns('', *urlpatterns)
Except ImportError: 
    pass
```